### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by rowanmanning**
Made some improvements based on our [writing style guidelines](https://github.com/Financial-Times/ft-origami#style):
* Remove use of the passive voice
* Simplify and split up some complex sentences
* Capitalise "Build Service"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -235,7 +239,7 @@ doPromise1(arg1)
 
 Instead of nesting callbacks you chain `.then()` calls. Callback in `then` may return (promise of) result for the next `then` callback.
```
Comment (alicebartlett) [inline README.md]: (I know this PR doesn't modify this line, but since you're improving the readability...) I think this line has a typo?

File: README.md [hunk 2]
```diff
@@ -83,31 +87,31 @@ make attach-dev
...
   * `GITHUB_PASSWORD`: The GitHub password corresponding to `GITHUB_USERNAME`.
-  * `METRICS_ENV`: Which environment to store metrics under. This defaults to `NODE_ENV`, which allows us to measure QA/production metrics separately.
+  * `METRICS_ENV`: The environment to store metrics under. This defaults to `NODE_ENV`, which allows us to measure QA/production metrics separately.
 
 
 Testing
 -------
 
-The tests are split into unit tests, integration tests, and an older suite of tests that we're in the process of migrating. To run tests on your machine you'll need to install [Node.js] and run `make install`. Then you can run the following commands:
+We split the tests into unit tests, integration tests, and an old suite of tests that we're migrating. To run tests on your machine you'll need to install [Node.js] and run `make install`. Then you can run the following commands:
```
Comment (alicebartlett) [inline README.md]: I think "the tests are split" is easier to understand here 😬

File: README.md [hunk 3]
```diff
@@ -127,7 +131,7 @@ Deployment
 
 The [production][heroku-production] and [QA][heroku-qa] applications run on [Heroku]. We deploy continuously to QA via [CircleCI][ci], You should never really deploy to QA manually. ~~We use a [Heroku pipeline][heroku-pipeline] to promote QA deployments to production~~.
```
Comment (alicebartlett) [inline README.md]: Capital "Y" on you there. Reword this as "you should never need to deploy to QA manually"

File: README.md (hunk 1 above)
Comment (rowanmanning) [inline README.md]: How about this?

> Instead of nesting callbacks, you chain `.then()` calls. Callbacks in `then` may return (a promise of) the result for the next `then` callback.

**Comment by alicebartlett** [comment]
All good! One case where I think a passive voice is easier to understand, and two changes to lines that you didn't even touch I just think they should be updated anyways.

File: README.md (hunk 1 above)
Comment (alicebartlett) [inline README.md]: yrp
